<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="TempAPI" tests="2" time="4.304">
  <testsuite name="Upload Multiple with one wrong file" id="1beeb3cb-0cbe-4d66-bbd2-77cea466add5" timestamp="2023-07-06T23:48:00.812Z" tests="1" failures="1" errors="0" time="4.220">
    <testcase name="File name check" time="4.220" classname="TempApi">
      <failure type="AssertionFailure" message="expected undefined to deeply equal '16475687531_eac8a30914_b.jpeg'">
        <![CDATA[Failed 1 times.]]>
        <![CDATA[Collection JSON ID: 1faea77b-c035-4129-a215-3608487d19b3.]]>
        <![CDATA[Collection name: TempAPI.]]>
        <![CDATA[Request name: Upload Multiple with one wrong file.]]>
        <![CDATA[Test description: File name check.]]>
        <![CDATA[Error message: expected undefined to deeply equal '16475687531_eac8a30914_b.jpeg'.]]>
        <![CDATA[Stacktrace: AssertionError: expected undefined to deeply equal '16475687531_eac8a30914_b.jpeg'
   at Object.eval sandbox-script.js:1:1).]]>
      </failure>
    </testcase>
  </testsuite>
  <testsuite name="Given_AnonUser_Unable_To_Upload" id="41275d63-ac81-4c9d-8241-0edc3e08d70b" timestamp="2023-07-06T23:48:00.812Z" tests="1" failures="0" errors="0" time="0.084">
    <testcase name="Status code should be 401" time="0.084" classname="TempApi"/>
  </testsuite>
</testsuites>